ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/filter-column.component.ts

```diff
@@ -87,4 +87,8 @@ export class FilterColumnComponent implements OnInit {
     this.dataFilter.notEmpty = false;
     this.dataFilter.empty = false;
   }
+
+  hideCheckbox(dataFilter: Filter): boolean {
```

Comment:
Let's rename this method so it's really clear that this is a bit of a hack.  How about `hideCheckboxIfRGPR21` or some such?  Or write a method comment to this effect.

Reply:
Suggested name is self-explanatory. I like it.

Could this be a reusable function which could be applied to other similar requirements in future. If yes, how about this change?

```
hideCheckboxIfThisColumn(dataFilter: Filter, name: string): boolean {
    return dataFilter?.participantColumn?.name === name;
  }
 ```